snippets/omt.snippets.json

```diff
@@ -39,6 +39,19 @@
             "             ${0:@LOG('snippet');}"
         ]
     },
+    "Declared imports": {
+        "prefix": "declare",
+        "description": "Declare modules with their imports.\n\nUse in OMT modules only.",
+        "body": [
+            "declare:",
+            "    ${1:ModuleName}:",
+            "        ${2:ThingToDeclare}:",
+            "            type: ${3|Activity,Command,Procedure|}",
```

Comment:
We should add 'Query' here, because it has been added. But it also needs to be added in the syntax highlighting definitions then, because that will not be recognised yet. Would be nice if that can be included in this PR, or we could do it in a separate one.

Reply:
I added this in 95697c1 :+1: